src/d3a/d3a_core/export.py

```diff
@@ -82,6 +82,11 @@ def __init__(self, root_area: Area, path: str, subdir: str, file_stats_endpoint,
             self.directory = pathlib.Path(self.rootdir, subdir)
             self.zip_filename = pathlib.Path(self.rootdir, subdir + "_results")
             mkdir_from_str(str(self.directory))
+            import d3a.constants
```

Comment:
I think you can move this to the top and also do it like:
`from d3a.constants import D3A_TEST_RUN`

Reply:
He is actually obliged to import the module and not the variable, because if the variable is imported at the beginning of the file, the value of the variable will remain the same even if its value is modified in the `before_all` function. By importing the module reference we can access the current value of the variable after we change it.